LedRGB.cpp

```diff
@@ -9,15 +9,21 @@ LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
 }
 
 void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
-    RedLed.setBrightness(r / 2.55);
-    GreenLed.setBrightness(g / 2.55);
-    BlueLed.setBrightness(b / 2.55);
+    r /= 2.55;
+    g /= 2.55;
+    b /= 2.55;
+    if (checkBrightness) {
```

Comment:
This does not work, please think about why.
a red led at 100% brightness would be 255,0,0.
Now imagine setting the brightness to 50%, what would the values need to be?
What do they get set to when using `setBrightness` as it is currently implemented?